src/Microsoft.OpenApi/Writers/OpenApiWriterExtensions.cs

```diff
@@ -181,25 +180,6 @@ public static void WriteProperty<T>(this IOpenApiWriter writer, string name, T v
             }
         }
 
-        /// <summary>
-        /// Write the optional of collection string.
-        /// </summary>
-        /// <param name="writer">The Open API writer.</param>
-        /// <param name="name">The property name.</param>
-        /// <param name="elements">The collection values.</param>
-        /// <param name="action">The collection string writer action.</param>
-        public static void WriteOptionalCollection(
```

Comment:
I am curious as to how we were able to remove this.  Isn't Tags an optional collection of strings?

Reply:
Actually Tags uses the generic `WriteOptionalCollection<T>` method to be written out. I think the thought process at the time was we could reuse the generic method to write out all types. Let me add it back.

Reply:
Tags are defined as an `IList<OpenApiTag>` hence they use the generic `WriteOptionalCollection<T>` method during serialization. However, security schemes and enums are collections of type string so I can add the method back.